### PR TITLE
Fieldset operations added

### PR DESCRIPTION
**Pull request by danielnaab**
#227 Add operations to the fieldset edit UI:

- "add question" implemented in the empty state and at the bottom of the fieldset
- "remove section" implemented in the empty state

### CODE REVIEW

File: packages/design/src/FormManager/FormEdit/PreviewPattern.tsx [hunk 1]
```diff
@@ -49,3 +49,26 @@ export const PreviewPattern: PatternComponent = function PreviewPattern(props) {
     </div>
   );
 };
+
+/**
+ * Determine if this mouse event corresponds to a pattern edit control, which
+ * should not trigger setting the pattern edit focus.
+ */
+const isPatternEditControlEvent = (
```
Comment (danielnaab) [inline packages/design/src/FormManager/FormEdit/PreviewPattern.tsx:57]: @natashapl This checks if we should treat the click event as something to change focus by looking for the `data-pattern-edit-control` attribute on the DOM node (or it's nearby parents).

File: packages/design/src/FormManager/FormEdit/components/FieldsetEdit.tsx [hunk 2]
```diff
@@ -48,16 +52,17 @@ const FieldsetPreview: PatternComponent<FieldsetProps> = props => {
...
-                  </a>
-                </span>
-                <span className="action-text remove-section display-inline-block text-top margin-right-2">
-                  <a className="usa-link" href="#">
-                    Remove section
-                  </a>
-                </span>
+                <div
+                  className="margin-right-2 bg-none"
+                  data-pattern-edit-control="true"
```
Comment (danielnaab) [inline packages/design/src/FormManager/FormEdit/components/FieldsetEdit.tsx:57]: This puts the drop-down out "out of bounds" for the edit focus handler.

File: package.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
     "format": "prettier --write \"packages/*/src/**/*.{js,jsx,ts,tsx,scss}\"",
     "lint": "turbo run lint",
     "pages": "rm -rf node_modules && npm i -g pnpm turbo && pnpm i && pnpm build && ln -sf ./apps/spotlight/dist _site",
-    "test": "vitest run --coverage.enabled --coverage.reporter=text --coverage.reporter=json-summary --coverage.reporter=json --coverage.reportOnFailure --reporter vitest-github-actions-reporter",
+    "test": "vitest run",
+    "test:ci": "vitest run --coverage.enabled --coverage.reporter=text --coverage.reporter=json-summary --coverage.reporter=json --coverage.reportOnFailure --reporter vitest-github-actions-reporter",
```
Comment (danielnaab) [inline package.json:16]: I split this out to speed up the pre-commit hook.

**Review by ethangardner — APPROVED** [review]
LGTM

**Pull request merged**